drivers/overlay/ov_serf.go

```diff
@@ -146,6 +146,7 @@ func (d *driver) processQuery(q *serf.Query) {
 		return
 	}
 
+	logrus.Debugf("Sending peer query resp %v", fmt.Sprintf("%s %s %s", peerMac.String(), net.IP(peerIPMask).String(), vtep.String()))
```

Comment:
IMO, unnecessary casting make the code bloated.
It is a minor comment, you don't need to change it.